### PR TITLE
Fix broken link and add Flathub to README

### PR DESCRIPTION
**Pull request by jwrober**
No related issue.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,10 @@ Installation
 
 [![Get Freeciv21 from the Snap Store](https://snapcraft.io/static/images/badges/en/snap-store-white.svg)](https://snapcraft.io/freeciv21)
 
+### Flathub Edition
+
+Freeciv21 is also available on [Flathub](https://flathub.org/apps/net.longturn.freeciv21).
```
Comment (lmoureaux) [inline README.md:35]: What about using https://flathub.org/badges next to the Snap one?

Comment (jwrober) [reply README.md:35]: Oh, didn't know that existed. Good catch. I'll do another commit.

**Comment by jwrober** [comment]
Hopefully that looks better @lmoureaux

**Comment by lmoureaux** [comment]
Fantastic!

**Pull request merged**